python/capellaAPI/CapellaAPIRequests.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+# Generic/Built-in
+import requests
+import os
+import logging
+import pprint
+
+
+# Other Libs
+
+
+# Owned
+from capellaAPI.CapellaAPIAuth import CapellaAPIAuth
+from capellaAPI.CapellaExceptions import MissingBaseURLError, MissingAccessKeyError, \
+    MissingSecretKeyError, GenericHTTPError
+
+# EnvVars.py sets the environmental variables used here
+# If EnvVars.py does not exist, then we'll try the OS environment variables instead
+try:
+    import capellaAPI.EnvVars
+except ImportError:
+    pass
+
+__author__ = 'Jonathan Giffard'
+__copyright__ = 'Copyright 2021, Couchbase'
+__credits__ = ['Jonathan Giffard']
+__license__ = 'MIT License'
+__version__ = '0.3.1'
+__maintainer__ = 'Jonathan Giffard'
+__email__ = 'jonathan.giffard@couchbase.com'
+__status__ = 'Dev'
+
+# Handles talking to the endpoints over http 
+# and returns the response.
+# Assumes that the caller then deals with whatever
+# the response holds.
+
+
+class CapellaAPIRequests:
+
+    def __init__(self):
+        # handles http requests - GET , PUT, POST, DELETE
+        # to the Couchbase Cloud APIs
+        # Read the values from the environmental variables
+        if os.environ.get('api_base_url') is None:
```

Comment:
Think these were changed to UPPER_CASE in the previous PR, we'll need to update these to keep things working!